docs/conf.py

```diff
@@ -94,7 +93,12 @@
 
 
 html_theme = 'scanpydoc'
-html_theme_options = dict(navigation_depth=4, logo_only=True)  # Only show the logo
+html_theme_options = dict(
+    navigation_depth=4,
+    logo_only=True,
+    docsearch_index='scanpy',
+    docsearch_key='fa4304eb95d2134997e3729553a674b2',
```

Comment:
I think version needs to be passed, but when I try it locally I still seem to get duplicate results.

Reply:
It doesn’t, as documented here: https://icb-scanpydoc.readthedocs-hosted.com/en/latest/scanpydoc.theme.html#confval-docsearch_doc_version

Reply:
Must be something else. What’s “duplicated”? From different versions?

Reply:
Yeah, I was getting results from both latest and stable.